ui/DriverSetup.py

```diff
@@ -0,0 +1,32 @@
+import platform
+import os
+import pytest
+from selenium import webdriver
+
+
+class DriverSetup:
+
+    @pytest.yield_fixture(autouse=True)
+    def init_browser(self):
+        options = webdriver.ChromeOptions()
+        options.add_argument('--ignore-certificate-errors')
+        options.add_argument('--ignore-ssl-errors')
+        options.add_argument('start-maximized')
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        if platform.system() == "Darwin":
+            chromedriver = dir_path + "/webdrivers/chromedriver"
+            os.environ["webdriver.chrome.driver"] = chromedriver
+            self.driver = webdriver.Chrome(options=options, executable_path=chromedriver)
+        else:
+            chromedriver = dir_path + "/webdrivers/chromedriver_linux"
+            os.environ["webdriver.chrome.driver"] = chromedriver
+            self.driver = webdriver.Chrome(options=options, executable_path=chromedriver)
+        self.driver.implicitly_wait(10)
+        # self.driver.maximize_window()
+
+
+        yield  # everything after 'yield' is executed on tear-down
+
+        if (self.driver != None):
```

Comment:
Could you try to use WebDriverManager and see if it can replace this setup classhttps://github.com/SergeyPirogov/webdriver_manager?